torch/csrc/jit/ir.h

```diff
@@ -391,14 +392,23 @@ struct Node : public Attributes<Node> {
   Value * input(size_t i) {
     return inputs_.at(i);
   }
-  const Value * input(size_t i) const {
+  Value * input(size_t i) const {
     return inputs_.at(i);
   }
 
+  Value* input(Symbol name) const { throw std::runtime_error("NO!"); }
```

Comment:
Forgot to remove it, I was making sure I found all the other cases. Symbols are convertible to integers so removing it beforehand would cause unchanged ones to hit input(size_t)